video/out/meson.build

```diff
@@ -8,7 +8,7 @@ protocols = [[wl_protocol_dir, 'stable/presentation-time/presentation-time.xml']
              [wl_protocol_dir, 'staging/content-type/content-type-v1.xml'],
              [wl_protocol_dir, 'staging/fractional-scale/fractional-scale-v1.xml'],
              [wl_protocol_dir, 'staging/single-pixel-buffer/single-pixel-buffer-v1.xml'],
-             ['protocols', 'xx-color-management-v4.xml']]
+             [wl_protocol_dir, 'staging/color-management/color-management-v1.xml']]
```

Comment:
This needs to be version checked, see below `if features['wayland-protocols-1-..']`

Reply:
done